docs/CONTRIBUTING.md

```diff
@@ -194,11 +194,11 @@ methods we'd recommend for each operating system:
   > If you're missing a package, when you try to `cargo install` or `cargo
 build` later, you'll get an error that looks like this:
   >
-  >     ```
-  >     error: linking with `cc` failed: exit code: 1
-  >     [lots of output]
-  >     = note: /usr/bin/ld: cannot find -l[something]
-  >     ```
+  >```console
+  >error: linking with `cc` failed: exit code: 1
+  >[lots of output]
+  >= note: /usr/bin/ld: cannot find -l[something]
+  >```
```

Comment:
This code block is text output. My understanding is that console code blocks are specifically for commands that you enter into the console, no?

Reply:
I think the console also contains what is printed out in your console(stdout + stderr). But I've updated it.